docs/index.rst

```diff
@@ -11,5 +11,6 @@
    cellxgene_census_docsite_schema.md
    cellxgene_census_docsite_data_release_info.md
    python-api
-   R API <https://chanzuckerberg.github.io/cellxgene-census/r/index.html>
-   cellxgene_census_docsite_FAQ.md
\ No newline at end of file
+   cellxgene_census_docsite_r_tutorials.md
+   R API <https://chanzuckerberg.github.io/cellxgene-census/r/reference/index.html>
```

Comment:
Where does the `reference` come from?

Reply:
@ebezzi thanks for highlighting this -- one thing I've noticed repeatedly with pkgdown landing pages (such as [ours](https://chanzuckerberg.github.io/cellxgene-census/r/index.html)) is that it's super easy to overlook the "Reference" link on the top bar, since all the other links on the page are blue underlined except for that one. So I find myself often getting mentally stuck on the landing page (which is just a copy of README.md) for a few seconds, until I remember to look for that link. 

With the static HTML vignettes suggested here, we're only using the pkgdown site as the API reference, so I thought I'd just link directly and skip that roadbump. Open to alternative ideas though. cc @pablo-gar